editor.planx.uk/src/@planx/components/InternalPortal/Editor.tsx

```diff
@@ -28,13 +28,15 @@ const InternalPortalForm: React.FC<{
   handleSubmit?: (val: any) => void;
   flows?: Array<Flow>;
   tags?: NodeTag[];
+  disabled?: boolean;
 }> = ({
   handleSubmit,
   text = "",
   flowId = "",
   flows = [],
   tags = [],
   notes = "",
+  disabled = false,
```

Comment:
We shouldn't need to define a fallback value like this at a low level - we should be able to do it once at a high level where we run our permission check. This goes hand-in-hand with making `disabled` a mandatory field and not an optional one 👍